File: backend/check_dbs.py
import sqlite3
import sys

def check_db(db_path):
    print(f"--- {db_path} ---")
    try:
        conn = sqlite3.connect(db_path)
        cursor = conn.cursor()
        
        cursor.execute("SELECT id, username, email, role FROM users WHERE role='admin'")
        admins = cursor.fetchall()
        print(f"Admins: {admins}")
        
        cursor.execute("SELECT COUNT(*) FROM lessons")
        lessons = cursor.fetchone()[0]
        print(f"Lessons: {lessons}")
        
        cursor.execute("SELECT COUNT(*) FROM club_events")
        events = cursor.fetchone()[0]
        print(f"Events: {events}")
        
        conn.close()
    except Exception as e:
        print(f"Error: {e}")

check_db('cyberclub.db')
check_db('cyberclub2.db')

File: backend/check_dbs.js
const sqlite3 = require('sqlite3').verbose();
const db1 = new sqlite3.Database('./cyberclub.db');
const db2 = new sqlite3.Database('./cyberclub2.db');

function queryDb(db, name) {
    db.serialize(() => {
        db.all("SELECT id, username, role FROM users WHERE role='admin'", (err, rows) => {
            console.log(name, 'Admins:', rows);
        });
        db.get("SELECT count(*) as count FROM lessons", (err, row) => {
            console.log(name, 'Lessons:', row ? row.count : 0);
        });
        db.get("SELECT count(*) as count FROM club_events", (err, row) => {
            console.log(name, 'Events:', row ? row.count : 0);
        });
    });
}
queryDb(db1, 'cyberclub.db');
queryDb(db2, 'cyberclub2.db');
